C/wrappedJets/wrappers.h

```diff
@@ -1,17 +1,35 @@
-#ifndef WRAPPERS_H
-#define WRAPPERS_H
+#ifndef WRAPPEDJETS_WRAPPERS_H
+#define WRAPPEDJETS_WRAPPERS_H
+
+#include "simplicity_assert.h"
```

Comment:
```suggestion
#include "../simplicity_assert.h"
```
Doesn't compile for me otherwise.